src/py/flwr/simulation/run_simulation.py

```diff
@@ -333,14 +328,10 @@ def _main_loop(
     try:
         # Register run
         log(DEBUG, "Pre-registering run with id %s", run.run_id)
-        init_status = RunStatus(Status.RUNNING, "", "")
-        state_factory.state().run_ids[run.run_id] = RunRecord(  # type: ignore
-            run=run,
-            status=init_status,
-            starting_at=now().isoformat(),
-            running_at=now().isoformat(),
-            finished_at="",
-        )
+        run.status = RunStatus(Status.RUNNING, "", "")
+        run.starting_at = now().isoformat()
+        run.running_at = now().isoformat()
```

Comment:
```suggestion
        run.starting_at = now().isoformat()
        run.running_at = run.starting_at
```
Should we make this equal?

Reply:
@panh99 @chongshenng does it make sense to have the same timestamp for both?